.github/scripts/plugins.py

```diff
@@ -165,8 +165,10 @@ def get_core_plugins() -> None:
     for plugin in CORE_PLUGINS:
         plugin["slug"] = "Plugins/" + plugin["name"].replace(" ", "+")
 
-    assert match  # Needed to stop match[1] giving 'error: Value of type "Optional[Match[str]]" is not indexable'
-    new_contents = contents.replace(match[1], template.render(plugins=CORE_PLUGINS))
+    # Needed to stop match[1] giving 'error: Value of type "Optional[Match[str]]" is not indexable'
```

Comment:
I can't figure out what's changed in these 4 lines and why...

One thing to say about the comment is that it was intentionally on the end of the assert line, to avoid the comment getting separated from the assert, which is easy to happen over time, under maintenance...

Reply:
Yes, the format-on-save autopep8 (I think? black is even more "rigorous") thought it would be a good idea to move the comment on top and the function arguments to a new line. Which also exlains the other changes. I've now taught VSCode how to forget about that and reverted these changes.

Reply:
Thank you!

Funnily enough, I've been working in another repo (an Obsidian plugin) recently that is very opinionated about things like auto-formatting - and what I like about it's setup is that it is thoroughly enforced via configuration in the repo, so I didn't need to do anything for my IDE...

That's probably a step too far for this repo right now, I feel... Baby steps...